rnambaale/project-board-backend
# reset password route

#### What does this PR do?
- allow users to reset their passwords
#### Description of Task to be completed?
- send reset password link to a verified user's email
- use the link in the email to allow users to set new passwords
#### How should this be manually tested?
 - locally, clone this repo and check into `reset-password-route` branch. 
- start the dev server with `npm run start:dev`. In postman, use the route `http://localhost:5000/api/v1/auth/reset-password` with the sample body ` {  "email": "xx@xxx.xxx" }`
- you should recieve an email in the provided email if it is registered. Copy the provided link and paste it as a `url` in postman. Using this format `{"password":"xxxx"}`. Send the request to successfully reset the password.
